template/{{.input_root_dir}}/{{template `project_name_alphanumeric_underscore` .}}/feature_engineering/notebooks/GenerateAndWriteFeatures.py.tmpl

```diff
@@ -116,11 +116,11 @@ features_df = compute_features_fn(
 # COMMAND ----------
 # DBTITLE 1, Write computed features.
 
+{{- if (eq .input_include_models_in_unity_catalog "no") }}
```

Comment:
Is there a reason you changed this from your last commit?
```suggestion
{{- if (eq .input_include_models_in_unity_catalog `no`) }}
```

Reply:
There were some failed tests and I noted previous conditionals in section above related to dbutils.wigets with `{{- if (eq .input_include_models_in_unity_catalog "no") }}`  so thought to replace... should it be `'no'`  instead of `"no"` ?

Reply:
Hmm generally we use backticks (`) instead of quotes (") or ('). You can see other examples here: https://github.com/search?q=repo%3Adatabricks%2Fmlops-stacks%20input_include_models_in_unity_catalog%20&type=code

Reply:
It's inconsistent within the existing repo. I will use backticks in my updates :)